chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/UserManagementController.java

```diff
@@ -337,7 +338,9 @@ public UserInvitation inviteUser(@RequestBody final UserInvitation invitation)
                 + "which will allow you to manage certified product listings on the CHPL. "
                 + "Please click the link below to create or update your account: <br/>"
                 + env.getProperty("chplUrlBegin") + "/#/registration/create-user/" + createdInvite.getInviteToken()
-                + "</p>" + "<p>If you have any questions, please contact the ONC CHPL Team at onc_chpl@hhs.gov.</p>"
+                + "</p>"
+                + "<p>If you have any issues completing the registration, "
+                + "please contact the ONC CHPL Team at onc_chpl@hhs.gov.</p>"
```

Comment:
These _are_ HTML emails; is it worth it to make these mailto: links?

Reply:
We can if you want. I've only ever sent these emails to my Ai account and looked at them in Outlook but in there they appear as mailto links anyway. I think it recognizes email addresses and displays them as such.

Reply:
Hmm. I hadn't noticed that. I forwarded one to my gmail account and it does that as a link too. I think it'd be a good idea to be explicit about it, but could go either way